zeppelin-zengine/pom.xml

```diff
@@ -625,6 +625,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>org.eclipse.jgit</groupId>
+        <artifactId>org.eclipse.jgit</artifactId>
+        <version>4.3.1.201605051710-r</version>
```

Comment:
can we use `${eclipse.jgit.version}` instead?

Reply:
Unfortunately, version `4.1.1.201511131810-r` which is the value of `${eclipse.jgit.version}` does not contain some classes such as `org.eclipse.jgit.api.RemoteAddCommand`. If possible, I can change the value of `eclipse.jgit.version` to use `4.3.1.201605051710-r` instead of `4.1.1.201511131810-r`. What do you think?

Reply:
I think we'd better upgrade the version of `jgit` to 4.3.x